modules/signals/rxjs-interop/src/rx-method.ts

```diff
@@ -42,8 +42,20 @@ export function rxMethod<Input>(
       return { destroy: noop };
     }
 
+    const callerInjector = getCallerInjector();
+    if (config?.injector === undefined && callerInjector === undefined) {
```

Comment:
For consistency with other warnings and because of tree shaking, let's also display this warning only in development mode:

```ts
declare const ngDevMode: unknown;

// ...
if (
  typeof ngDevMode !== 'undefined' &&
  ngDevMode &&
  config?.injector === undefined &&
  callerInjector === undefined
) {
```